docs/auto-splitters.md

```diff
@@ -286,6 +286,42 @@ end
 
         * Cheat Engine is a tool that allows you to easily find Addresses and Pointer Paths for those Addresses, so you don't need to debug the game to figure out the structure of the memory.
 
+## sig_scan
+sig_scan performs a signature/pattern scan on the provided IDA-style byte array and optional integer offset and returns the found address
+
+Example:
+`signature = sig_scan("89 5C 24 ?? 89 44 24 ?? 74 ?? 48 8D 15", 4)`
+
+Returns:
+`14123ce19`
+
+* Note: Until the address is found, sig_scan returns a 0.
+* Note: Signature scanning is an expensive action. So, once an address has been found, it's recommended to reassign the sig_scan variable with the result of the sig_scan function to stop the scanning.
+
+
+Mini example script with the game SPRAWL:
+```lua
+process('Sprawl-Win64-Shipping.exe')
+
+local featuretest = 0
+
+function state()
+    -- Perform the signature scan to find the initial address
+    featuretest = sig_scan("89 5C 24 ?? 89 44 24 ?? 74 ?? 48 8D 15", 4)
+
+    if featuretest == 0 then
```

Comment:
Same problem here, this will NEVER be true (if sig_scan returns "0")
![image](https://github.com/user-attachments/assets/caef001d-0c6b-4a7c-a9e7-10d8feb78e56)